tsdb/agent/db_test.go

```diff
@@ -45,10 +45,10 @@ func TestUnsupported(t *testing.T) {
 	logger := log.NewNopLogger()
 
 	s, err := Open(logger, prometheus.DefaultRegisterer, nil, promAgentDir, opts)
-	if err != nil {
-		t.Fatalf("unable to create storage for the agent: %v", err)
-	}
-	defer s.Close()
+	require.NoError(t, err)
```

Comment:
It can be assumed that this error was because the storage couldn't be created, so I went ahead and simplified the error checks.